chaos.py

```diff
@@ -28,10 +28,36 @@
 import encryption  # noqa: F401
 
 
+class LessThanFilter(logging.Filter):
+    """
+    Source: https://stackoverflow.com/questions/2302315/how-can-info-and-
+    debug-logging-message-be-sent-to-stdout-and-higher-level-messag
```

Comment:
Change this to just https://stackoverflow.com/questions/2302315 to avoid wrapping a link